### PR TITLE
Add acq-operation.json schema and example file

### PR DESCRIPTION
**Pull request by J-Oliveros**

### CODE REVIEW

File: schema/acq-operation.json [hunk 1]
```diff
@@ -1,10 +1,71 @@
...
+      "$ref": "./identifier.json",
+      "description": "unique identification of the operation"
+    },
+    "status": {
+      "type": "string",
+      "description":  "status of the data acquisition"
+    },
+    "operationType": {
+      "type": "string",
+      "description":  "collection technique for the operation"
```
Comment (jwaspin) [inline schema/acq-operation.json:32]: Needs enum with the allowed values (this list comes from the MI_OperationTypeCode code list)

File: schema/acq-operation.json [hunk 2]
```diff
@@ -1,10 +1,71 @@
...
+      "$ref": "./citation.json",
+      "description": "identification of the mission"
+    },
+    "identifier": { 
+      "$ref": "./identifier.json",
+      "description": "unique identification of the operation"
+    },
+    "status": {
+      "type": "string",
+      "description":  "status of the data acquisition"
```
Comment (jwaspin) [inline schema/acq-operation.json:28]: Needs enum with the allowed values (this list comes from the MD_ProgressCode code list)

**Comment by jwaspin** [comment]
@dwalt Is https://standards.iso.org/iso/19115/resources/Codelists/gml/ a better source? Or where should we be referring to double check these are up-to-date?

**Comment by hmaier-fws** [comment]
> @dwalt Is https://standards.iso.org/iso/19115/resources/Codelists/gml/ a better source? Or where should we be referring to double check these are up-to-date?

@jwaspin @J-Oliveros I believe the authoritative source for all things ISO (outside of the published standards) are: https://standards.iso.org/ and https://schemas.isotc211.org/ @dwalt can verify or refute that assertion. The https://schemas.isotc211.org/ site provides links to several different views of the standards, which can be helpful, for example, viewing the "schema documentation" for [`MI_AcquisitionInformation`](https://schemas.isotc211.org/19115/-2/gmi/1.0/gmi/#element_MI_AcquisitionInformation) is often easier than attempting to decode the [acquisitionInformation.xsd](https://standards.iso.org/iso/19115/-2/gmi/1.0/acquisitionInformation.xsd) on the "standards" site.

File: schema/acq-operation.json [hunk 3]
```diff
@@ -1,10 +1,101 @@
...
+    "childOperation": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "sub-missions that make up part of a larger mission"
+    },
+    "plan": {
+      "type": "string",
+      "description": "plan satisfied by the operation"
```
Comment (hmaier-fws) [inline schema/acq-operation.json:84]: @dwalt is it correct that this should be a string? According to [MI_AcquisitionInformation](https://schemas.isotc211.org/19115/-2/gmi/1.0/gmi/#element_MI_AcquisitionInformation) it seems that `acquisitionPlan` should be a class of type [MI_Plan](https://schemas.isotc211.org/19115/-2/gmi/1.0/gmi/#element_MI_Plan)

**Comment by dwalt** [comment]
@jwaspin the ISO codelist is better than ESIP. The TC211 is probably the best source, but hard to read. In the ISO requirement documents, I identified codelist changes, though I have not done that analysis with 19157 2023.

File: schema/acq-operation.json (hunk 3 above)
Comment (dwalt) [reply schema/acq-operation.json:84]: @jwaspin The Operation class references a single Plan instance using an identifier: https://github.com/adiwg/mdJson-schemas/issues/217

**Review by jwaspin — CHANGES_REQUESTED** [review]
Change plan to $ref to the acq-plan class

**Review by jwaspin — APPROVED** [review]

**Pull request merged**